File: hello/go/hello.go
package hello

import (
	"fmt"
	"net/http"
)

func init() {
	http.HandleFunc("/", main)
}

func main(w http.ResponseWriter, r *http.Request) {
	fmt.Fprintf(w, "<h1 style='text-align: center;'>Hello Go</h1><p style='text-align: center;'>Welcom to Cloud APP World!</p>");
}

File: hello/python/hello.py
import webapp2

class MainPage(webapp2.RequestHandler):
    def get(self):
        self.response.out.write("<h1 style='text-align: center;'>Hello Python</h1><p style='text-align: center;'>Welcom to Cloud APP World!</p>")

app = webapp2.WSGIApplication([('/', MainPage),], debug=True)
